tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -126,13 +146,21 @@ Object.Dai {
 		sample_bits		32
 		quirks			"lbm_mode"
 		io_clk		$MCLK
+		version         $SSP_BLOB_VERSION
 
 		Object.Base.hw_config.1 {
 			name	"SSP2"
 			id	0
 			mclk_freq	$MCLK
 			bclk_freq	3072000
 			tdm_slot_width	32
+			IncludeByKey.PLATFORM {
+				"mtl" {
+					Object.Base.link_config.1 {
+						clock_source	2
+					}
+				}
+			}
```

Comment:
@juimonen can we take this from platform.conf i.e. using the same method as ```SSP_BLOB_VERSION``` above ? This could be an attribute in teh base class that would default to ```PLATFORM_CLOCK_blah_DEFAULT``` ?

Reply:
I don't currently know a way to add this to all hw_configs in a certain platform in some other way than this